src/shared/components/input/Option.tsx

```diff
@@ -0,0 +1,6 @@
+interface Option {
+  label: string
+  value: string
+}
+
+export type { Option }
```

Comment:
I know that this is a remnant of the previous `SelectWithLabelFormGroup` code, but it might be a good idea to change the name of this interface while we still have the chance...`Option` is quite ambiguous